ansible-ipi-install/roles/post-install/tasks/main.yml

```diff
@@ -0,0 +1,98 @@
+- name: Wait 7m for all the nodes to settle
+  command: sleep 900
+
+- block:
+  - name: Label infra nodes
+    shell: |
+       oc label node '{{ infra_node_string_1 }}' 'node-role.kubernetes.io/infra=' --overwrite 
+
+  - name: Label infra nodes
+    shell: |
+       oc label node '{{ infra_node_string_2 }}' 'node-role.kubernetes.io/infra=' --overwrite
+
+  - name: Label workload nodes
+    shell: |
+       oc label node '{{ workload_node_string }}' 'node-role.kubernetes.io/workload=' --overwrite
+  when: infra_node_string_1 != "" or infra_node_string_2 != "" or workload_node_string != ""
+
+- name: OVN block
+  block:
+  - name: Scale CVO Down for OVNKubernetes image patching
+    command: oc scale -n openshift-cluster-version deployment.apps/cluster-version-operator --replicas=0
+  
+  - name: Patch new OVNKubernetes Image
+    command: oc -n openshift-network-operator set env deployment.apps/network-operator OVN_IMAGE={{openshift_ovn_image}} RELEASE_VERSION="5.0.0"
+  
+  - name: Pause the rollout to begin
```

Comment:
Hey @mohit-sheth , mind you update the OVN image rollout logic to be aligned with the latest changes in scale-ci-deploy¿